plugins/woocommerce/client/blocks/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md

```diff
@@ -28,44 +28,44 @@ The majority of our feature flagging is blocks, this is a list of them:
 
 - Clear (Experimental)
```

Comment:
A lot of these links are either incorrect because the directories have changed (e.g. `/blocks/assets/js/blocks/product-filter/inner-blocks/stock-filter/index.ts`) or are no longer experimental.

Reply:
Nice catch! I think it would be best to audit those separately, though. Fixing 404 links makes sense in the context of this PR, but it feels out of scope. I'll create an issue for it.